test/UnitTests/Microsoft.Testing.Platform.UnitTests/.editorconfig

```diff
@@ -0,0 +1,29 @@
+[*.cs]
+
+# VSTHRD200: Use "Async" suffix for async methods
+dotnet_diagnostic.VSTHRD200.severity = none
+
+# CA1822: Mark members as static
+dotnet_diagnostic.CA1822.severity = suggestion
+
+# CA1707: Identifiers should not contain underscores
+dotnet_diagnostic.CA1707.severity = suggestion
+
+# SA1300: Element should begin with upper-case letter
+dotnet_diagnostic.SA1300.severity = none
+
+# SA1203: Constants should appear before fields
+dotnet_diagnostic.SA1203.severity = none
+
+# SA1009: Closing parenthesis should be spaced correctly
+dotnet_diagnostic.SA1009.severity = none
+
+# SA1111: Closing parenthesis should be on line of last parameter
+dotnet_diagnostic.SA1111.severity = none
+
+# SA1414: Tuple types in signatures should have element names
+dotnet_diagnostic.SA1414.severity = none
+
+dotnet_diagnostic.CA1861.severity = none
+
+dotnet_diagnostic.CA1001.severity = none
```

Comment:
I feel like this one should not be disabled but instead violations should be fixed.

Reply:
🤷‍♀️ it's the exact same code as in the original repo, so either we are not reporting it there or we also ignore them, or there were changes that I did not mean to do.

Reply:
This is what it is complaining about: 

test\UnitTests\Microsoft.Testing.Platform.UnitTests\ServerMode\ServerDataConsumerServiceTests.cs(19,21): error CA1001: Type 'ServerDataConsumerServiceTests' owns disposable field(s) '_service' but is not disposabl
e (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1001) [S:\p\testanywhere\mstest\test\UnitTests\Microsoft.Testing.Platform.UnitTests\Microsoft.Testing.Platform.UnitTests.csproj::TargetFramework=net462]

can we merge this now, and discuss later?